Client/game_sa/CPoolsSA.cpp

```diff
@@ -1419,3 +1420,42 @@ int CPointerNodeSingleLinkPoolSA::GetNumberOfUsedSpaces()
 
     return iOut;
 }
+
+unsigned int CPoolsSA::AllocateTextureDictonarySlot(uint uiSlotId, std::string& strTxdName)
```

Comment:
Perhaps use `const char*` instead of `std string`?

Reply:
Any reason?

Reply:
std::string& can be replaced with string_view here

Reply:
You are right